_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,290 @@
+- Change Name: rfc-for-rfcs
+- Start Date: 2017-11-03)
+- RFC PR: (leave this empty)
+- Core-Plans Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC is meant to serve as an example of the process for entering RFCs. It should also serve up the information regarding the RFC process that will get merged into a .md file in the repository for explanation of the why/when/how of an RFC against core-plans.
+
+# Motivation
+[motivation]: #motivation
+
+We want to provide safety-nets and transparency over core-plans direction and development efforts
+while also enabling new contributors to feel comfortable in anything they might want to add to the 
+core package set.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The RFC for RFC Implementation adds a clearly defined process for _when_, _why_ ,  and _how_ to open RFCs for changes in the core-plans repository. 
+
+The expected impact is that core-plans maintainers have another tool in the maintenance process toolchest to enable them to deal with tricky changes, whether those are substantial in scope, or substantial in outcome.
+
+Rather than providing an example-driven introduction to the RFC process this entire RFC exists as an example of the RFC process. The explicit/reference level explanation contains a complete account of the RFC process top to bottom.
+
+
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+#################
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Intro](#core-plans-rfcs)
+  - [Table of Contents](#table-of-contents)
+  - [When you need to follow this process](#when-you-need-to-follow-this-process)
+  - [Before creating an RFC](#before-creating-an-rfc)
+  - [What the process is](#what-the-process-is)
+  - [The RFC life-cycle](#the-rfc-life-cycle)
+  - [Reviewing RFCs](#reviewing-rfcs)
+  - [Implementing an RFC](#implementing-rfcs)
+  - [RFC Postponement](#rfc-postponement)
+  - [Help this is all too informal!]
+  - [Attribution]
+
+# Core-Plans RFCs
+[Core-Plans RFCs]: #core-plans-rfcs
+
+A majority of contributor and maintainer work can be implemented and reviewed via the normal GitHub pull request workflow. These changes include bug fixes, documentation improvements, package version bumps, and adding a new core-plan. 
```

Comment:
Would it be a good idea to link to the GitHub pull request flow for the unfamiliar? https://guides.github.com/introduction/flow/